src/coreclr/vm/virtualcallstub.cpp

```diff
@@ -619,24 +564,15 @@ void VirtualCallStubManager::Init(BaseDomain *pDomain, LoaderAllocator *pLoaderA
             if (dwWastedReserveMemSize != 0)
             {
                 DWORD cWastedPages = dwWastedReserveMemSize / GetOsPageSize();
-                DWORD cPagesPerHeap = cWastedPages / 6;
-                DWORD cPagesRemainder = cWastedPages % 6; // We'll throw this at the resolve heap
+                DWORD cPagesPerHeap = cWastedPages / 2;
+                DWORD cPagesRemainder = cWastedPages % 2; // We'll throw this at the cache entry heap
```

Comment:
Some clarifying comments on the value `2` would be helpful. Is it possible we can compute this based on the `GetOsPageSize()` or is this simply a value we use for an optimization? We converted from 4k to 16k, factor of 4, but we changed this by a factor 3 so that is where I am coming from.

Reply:
This is about going from having 6 LoaderHeap structures as part of a VirtualCallStubManager to only having 2.